master/internal/trial.go

```diff
@@ -210,7 +210,7 @@ type trial struct {
 	lastContainerConnectedTime time.Time
 	startedContainers          map[cproto.ID]bool
 	containers                 map[cproto.ID]cproto.Container // only for running containers.
-	containerRanks             map[cproto.ID]int              // only for running containers.
+	containerRanks             map[cproto.ID]int              // only for started containers.
```

Comment:
This isn't true if a container fails to start. we have a `startedContainers` that is based on the container states. We probably should just remove the comment here and move this line to right below `allocations`.

Reply:
Updated to indicate this is for launched containers.